template/Makefile.j2

```diff
@@ -34,7 +33,7 @@ crds: deploy/helm/{[ operator.product_string }]-operator/crds/crds.yaml
 
 deploy/helm/{[ operator.product_string }]-operator/crds/crds.yaml:
 	mkdir -p deploy/helm/{[ operator.product_string }]-operator/crds
-	cat deploy/crd/*.yaml | yq e '.metadata.annotations["helm.sh/resource-policy"]="keep"' - > ${@}
+	cat deploy/crd/*.yaml | yq '.metadata.annotations["helm.sh/resource-policy"]="keep"' - > ${@}
```

Comment:
What is that doing? Is "eval" the default action? I'd be in favor of being explicit in scripts, because it's easier to read

Reply:
I'm starting to suspect that this is down to using two completely different scripts that are just both called `yq`. The one I'm using (Arch's `yq` package, seems to be https://github.com/kislyuk/yq) doesn't have subcommands at all, just like `jq` doesn't. `nixpkgs#yq` also seems to point towards the same source repository. Which one are you using?

Reply:
Oh! I'm using this one https://github.com/mikefarah/yq it's also the one that's used in the github actions. We should definitely pick one :smile:

Reply:
I'm using whatever `snap install yq` installed and I wrote the first version. Sorry.

Reply:
That is also the version I'm using, it references the github repo I linked as well

Reply:
I think since this is also running in the github actions, the mikefarah variant is probably the one we want to code against

Reply:
Looks like that one is packaged by Arch as `go-yq`, and Nixpkgs as `yq-go`, fwiw. Whichever one we end up using, it should probably also be documented somewhere.

Reply:
f8f71dc298962ae78264662209997c092bb4b460